cardano-api/src/Cardano/Api/Typed.hs

```diff
@@ -2378,26 +2260,12 @@ withNodeProtocolClient (CardanoMode epochSlots securityParam) f =
     f (mkNodeClientProtocolCardano epochSlots securityParam)
 
 
-data LocalNodeClientProtocols block =
-     LocalNodeClientProtocols {
-       localChainSyncClient
-         :: Maybe (ChainSyncClient
-                    block
-                    (Tip block)
-                    IO ())
-
-     , localTxSubmissionClient
-         :: Maybe (LocalTxSubmissionClient
-                    (GenTx block)
-                    (ApplyTxErr block)
-                    IO ())
-
-     , localStateQueryClient
-         :: Maybe (LocalStateQueryClient
-                    block
-                    (Query block)
-                    IO ())
-     }
+data LocalNodeClientProtocols block
+  = LocalNodeClientProtocols { localChainSyncClient :: Maybe (ChainSyncClient block (Tip block) IO ())
+                             , localTxSubmissionClient :: Maybe
+  (LocalTxSubmissionClient (GenTx block) (ApplyTxErr block) IO ())
+                             , localStateQueryClient :: Maybe (LocalStateQueryClient block (Query block) IO ())
+                             }
```

Comment:
Honestly this is much worse than before. Just look at it.

Reply:
I've disabled record formatting altogether.